src/Components/DefLinear/DefLinear.css

```
@@ -0,0 +1,15 @@
+.header {
+    font-size: 3em;
+    font-weight: bold;
+    color: #95CF67;
+}
+
+.Content {
+    font-size: 2em;
+    margin-left: 2em;
+    margin-right: 2em;
```

You can use some fancy CSS tricks here to keep the code tight:

```suggestion
    margin: auto 2em;
```